fe/fe-core/src/main/java/com/starrocks/connector/metadata/MetadataCollectJob.java

```diff
@@ -124,6 +124,7 @@ protected ConnectContext buildConnectContext(SessionVariable originSessionVariab
         context.setQueryId(UUIDUtil.genUUID());
         context.setExecutionId(UUIDUtil.toTUniqueId(context.getQueryId()));
         context.setStartTime();
+        context.setCurrentWarehouse(originSessionVariable.getWarehouseName());
```

Comment:
good job! I have a question. how do you build the iceberg distributed plan case to get here? Did you modify the session var `plan_mode=distributed` ?